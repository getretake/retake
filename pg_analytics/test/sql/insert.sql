CREATE TABLE t (a int, b int);
INSERT INTO t VALUES (1, 2);
CREATE TABLE s (a int, b int) USING deltalake;
INSERT INTO s SELECT * FROM t;
SELECT * FROM s;
DROP TABLE s, t;
