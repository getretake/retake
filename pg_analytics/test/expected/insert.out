CREATE TABLE t (a int, b int);
INSERT INTO t VALUES (1, 2);
CREATE TABLE s (a int, b int) USING deltalake;
INSERT INTO s SELECT * FROM t;
SELECT * FROM s;
 a | b 
---+---
 1 | 2
(1 row)

DROP TABLE s, t;
